Sources/PenguinStructures/Graphs/AdjacencyList.swift

```diff
@@ -61,6 +61,7 @@ public struct AdjacencyList<IdType: BinaryInteger>: GraphProtocol {
 		let offset: Int
 	}
 
+	/// Ensures `id` is a valid vertex in `self`; halts execution otherwise.
```

Comment:
This is very style-preference-nit-ish, so totally up to you, but:
```suggestion
	/// Ensures `id` is a valid vertex in `self`, halting execution otherwise.
```